RELEASES.md

```diff
@@ -0,0 +1,41 @@
+# Releases Guide
+
+Here, we will briefly understand the release plan and versioning steps. 
+This will cover the release process, major and minor releases, backporting the PR and manual release steps.
+
+If there is something that you require or this document leaves out, please
+reach out by [filing an issue](https://github.com/coredns/coredns/issues).
+
+## Releases
+
+Releases will be created from the release branches. The release will follow the following convention as `<major>.<minor>.<patch>`. 
+All the major and minor releases will be made from the master. The tag will be of the format `v<major>.<minor>.<patch>`.
```

Comment:
This is hard to follow:  

> Releases will be created from the release branches.

and 

> All the major and minor releases will be made from the master.

These seem to be directly contradictory.

Reply:
Done